### DISCUSSION TITLE
[WASM| Code breaking change when updating from 5.1.0-dev.1317 to 5.1.0-dev.1569

### DISCUSSION THREAD

**Discussion by DierkDroth** [unoplatform/uno#15216] [General]
@jeromelaban today the code breaking change below caused significant trouble: .MaxWidth=NaN now throws an exception. It took a while until we sorted the case which was exposed by our own regression tests and human testing as well.

It's my understanding that code breaking changes would be announced. Did we miss the announcement?

<img width="1280" alt="image" src="https://github.com/unoplatform/uno/assets/26179242/b46a25ec-15e6-4b3e-8018-8d763bb6c6fc">

**Comment by jeromelaban** [comment]
Thanks for the report @DierkDroth. You cannot miss an announcement that has not been made since it's in a preview build. We're only announcing changes in stable releases.

This is not a code breaking change, though, this is a behavior breaking change that results in the alignment to the WinUI behavior which does not accept `NaN` for those properties, introduced in https://github.com/unoplatform/uno/pull/15050.

**Comment by DierkDroth** [comment]
Thanks for your feedback @jeromelaban.

Couple of questions:
- you're saying the change was made to match the WinUI behavior. However, WinUI worked since day 1 with `.MaxWidth=NaN` in the template. No exception, nothing ... what am I missing?
- where would I find the list of code breaking changes for upcoming UNO 5.1?

**Reply by jeromelaban** [reply]
Thanks, we'll look into the behavior difference, there may be situations where the MaxWidth set to NaN may still be value.

As for the upcoming breaking changes, if notable, they're not listed at this time, aside from being in the commits themselves.

**Comment by DierkDroth** [comment]
Sounds good @jeromelaban
